ambry-api/src/main/java/com/github/ambry/config/NetworkConfig.java

```diff
@@ -37,6 +37,7 @@ public class NetworkConfig {
   public static final String SELECTOR_USE_DIRECT_BUFFERS = "selector.use.direct.buffers";
   public static final String REQUEST_QUEUE_TYPE = "request.queue.type";
   public static final String REQUEST_QUEUE_TIMEOUT_MS = "request.queue.timeout.ms";
+  public static final String REQUEST_QUEUE_CAPACITY = "request.queue.capacity";
```

Comment:
I think this a good config. If we want an unbounded queue, we can set it to a high val or -1, if we interpret -1 to be infinity ?